sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateRunDao.java

```diff
@@ -185,8 +185,11 @@ public Run getByAlias(String alias) throws IOException {
   @Override
   public List<Run> listByProjectId(long projectId) throws IOException {
     Criteria idCriteria = currentSession().createCriteria(Run.class, "r");
-    idCriteria.createAlias("r.containers", "container").createAlias("container.partitions", "partition");
-    idCriteria.createAlias("partition.pool", "pool").createAlias("pool.pooledElementViews", "dilution");
+    idCriteria.createAlias("r.containers", "container")
+        .createAlias("container.partitions", "partition")
+        .createAlias("partition.pool", "pool")
+        .createAlias("pool.poolDilutions", "poolDilution")
+        .createAlias("poolDilution.poolableElementView", "dilution");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "poolDilution.poolableElementView" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)